src/features/game/events/landExpansion/drillOilReserve.ts

```diff
@@ -0,0 +1,62 @@
+import Decimal from "decimal.js-light";
+import { GameState, OilReserve } from "features/game/types/game";
+import cloneDeep from "lodash.clonedeep";
+
+export type DrillOilReserveAction = {
+  type: "oilReserve.drilled";
+  id: string;
+};
+
+type Options = {
+  state: Readonly<GameState>;
+  action: DrillOilReserveAction;
+  createdAt?: number;
+};
+
+export const BASE_OIL_DROP_AMOUNT = 10;
+export const OIL_RESERVE_RECOVERY_TIME = 20 * 60 * 60 * 1000;
+
+export function canDrillOilReserve(
+  reserve: OilReserve,
+  now: number = Date.now()
+) {
+  return now - reserve.oil.drilledAt > OIL_RESERVE_RECOVERY_TIME * 1000;
```

Comment:
Looks like you're multiplying to 1000 twice, 
OIL_RESERVE_RECOVERY_TIME is already Milliseconds.

Reply:
Nice catch. I think this snuck back in while playing around with time while testing 😱